modules/ibm-z-secure-execution.adoc

```diff
@@ -101,4 +98,44 @@ base64 <your-hostkey>.crt
 +
 Compared to guests not running IBM Secure Execution, the first boot of the machine is longer because the entire image is encrypted with a randomly generated LUKS passphrase before the Ignition phase.
 
+. Add Ignition protection
++
+To protect secrets that are stored in the Ignition config file from being read or even modified the   Ignition config file must be encrypted.
++
+[NOTE]
+====
+To achieve the desired functionality, Ignition logging and local login are disabled by default when running IBM Secure Execution.
+====
+.. Fetch the public GPG key for the `secex-qemu.qcow2` image and encrypt the Ignition config with the key, by running the following command:
++
+[source,terminal]
+----
+gpg --recipient-file /path/to/ignition.gpg.pub --yes --output /path/to/config.ign.gpg --armor --encrypt /path/to/config.ign
+----
++
```

Comment:
before starting the VM `config.ign.gpg` should be attached with new `serial=ignition_crypted` :
```
-drive if=none,id=ignition,format=raw,file=config.ign.gpg,readonly=on -device virtio-blk,serial=ignition_crypted,drive=ignition
```